### PR TITLE
Extensions What's New updates for early April, 2022

### PR DESCRIPTION
**Pull request by dotproto**
Note that dynamic content scripts can now target the main world.

Address outstanding feedback from #2557.

### CODE REVIEW

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ updated: 2022-02-17
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### Chrome 102: {: #m102-registercontentscripts-main-world }
+
+April 14, 2022
+
+Dynamically registered content scripts can now specify a target world. See
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:23]: Is 'target world' something done in your product? If not, I would recommend replacing this a more easily understandable expression.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/whatsnew/index.md (hunk 1 above)
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:23]: Yep, it's an extension-specific concept. In short, extension content scripts normally run in a separate JS context from page script. We call this the extension's isolated world. The concept is explained in more detail in our [content scripts docs](https://developer.chrome.com/docs/extensions/mv3/content_scripts/#isolated_world).

I'm going to tweak the wording here to try to make it a little more clear and to link to that documentation.

Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md]: Thanks for the link.

**Review by jpmedley — APPROVED** [review]

File: site/en/docs/extensions/whatsnew/index.md [hunk 2]
```diff
@@ -16,6 +16,15 @@ updated: 2022-02-17
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### Chrome 102: {: #m102-registercontentscripts-main-world }
+
+April 14, 2022
+
+Dynamically registered content scripts can now specify the
```
Comment (Andarist) [inline site/en/docs/extensions/whatsnew/index.md:23]: Is it possible to declare a main world content script in the manifest? According to the available schema there is no such option: https://json.schemastore.org/chrome-manifest

**Pull request merged**